PolyEngine/Engine/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+find_package(OpenGL REQUIRED)
+if(WIN32)
+	find_package(GLEW REQUIRED PATHS ThirdParty/glew/lib/Release/ PATH_SUFFIXES Win32 x64) #FIXME(vuko): does this even work? how to make it search default paths first? (according to docs it should by default, but it fails on my system as opposed to invocation without given paths)
```

Comment:
Usually you have to set certain variable. I have and example here (https://github.com/MuniuDev/SpaceRush/blob/master/CMakeLists.txt) of working multi OS cmake with glew and other stuff.